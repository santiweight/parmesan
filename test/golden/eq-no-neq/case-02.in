module Foo where

instance Eq T1 where
  (==) = ...
  (/=) = ...

instance Eq T2 where
  x == y = ...
  x /= y = ...

instance Eq T3 where
  (Cons1 x) == (Cons2 y) = ...
  (Cons1 x) /= (Cons1 y) = ...
