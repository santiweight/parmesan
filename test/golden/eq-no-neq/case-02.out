module Foo where

instance Eq T1 where
  (==) = ...

instance Eq T2 where
  x == y = ...

instance Eq T3 where
  (Cons1 x) == (Cons2 y) = ...
